### ISSUE TITLE
Lazy load backgrounds thumbnails

### ISSUE THREAD

**Issue by m4dz** [cozy/cozy-home#585]
When opening the home dashboard, all backgrounds thumnails are loaded. We should lazy-load them to prevent bandwith extra-use (especially in mobile-use).

**Comment by clochix** [comment]
I strongly agree, loading Cozy takes way too much time when you have a slow connection.

**Comment by pierrevdk** [comment]
+1000, when I'm on my smartphone Cozy is almost impossible to use
